### ISSUE TITLE
outdated

### ISSUE THREAD

**Issue by alexaleluia12** [aws-samples/eb-demo-php-simple-app#4]
I am new to AWS world 
I tried to run this on eb and does not work

It seems that mysql can be installed direct but use RDS.
https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/using-features.managing.db.html

**Comment by jschwarzwalder** [comment]
@alexaleluia12 You are correct, this repository is 4 years old, and not being updated since 2016. 

What are you trying to accomplish?

**Comment by alexaleluia12** [comment]
@jschwarzwalder  I am trying to packer this code in a docker image and use elastic beanstalk.
This sample good because is code is simple and uses MySQL and portable due to docker.

**Comment by jschwarzwalder** [comment]
Have you looked into Electronic Container Storage? 

https://aws.amazon.com/ecs/

**Comment by jschwarzwalder** [comment]
There is also an entire section in the Elastic Beanstalk Developer Guide about using Docker: 

https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create_deploy_docker.html

If you find any errors in the guide, you can submit feedback, or create an issue on their GitHub repo.

**Comment by alexaleluia12** [comment]
Thanks @jschwarzwalder

**Comment by sergiomb2** [comment]
Failed to deploy application.

**Comment by elipwns** [comment]
Failed Environment update activity. Reason: Configuration validation exception: Invalid option specification (Namespace: 'aws:elasticbeanstalk:container:php:phpini', OptionName: 'document_root'): Unknown configuration setting.
